### PR TITLE
Tweak bone mutations descriptions

### LINKED ISSUE #25707: [BUG] Hollow/Light Bones mutation not conferring speed advantages.
The 'Light Bones' mutation and 'Hollow Bones' mutation advantages +10% speed and +20% speed respectively do not seem to work in conjunction with the 'quick' trait.

**To Reproduce**  
1. Make a player with the traits 'quick'.
2. Use the debug menu to add the bird mutation categories for 'hollow bones'.
3. Character @ only shows and confers the +10% speed bonus from quick, and not the ones from Light or Hollow Bones.

**Expected behavior**  
I expected either the 20% bonus from the hollow bones perk to take priority over quick, or stack, and give me a speed of either 120% if replacing, or 130% if stacking with quick.

**Versions and configuration(please complete the following information):**
 -Windows 7
 - Game Version [0.C-32238-g8f0b634]
 - Graphics version [Tiles]
 - Mods loaded [None]

**Additional context**

### PR DESCRIPTION
**Pull request by arijust**
#### Summary

```SUMMARY: Bugfixes "Tweak bone mutations descriptions."```
#### Purpose of change

Closes  #25707
#### Describe the solution

Write correct descriptions for what actually these mutations do.

### CODE REVIEW

**Comment by anothersimulacrum** [comment]
The fixed descriptions and the old descriptions have the same information, this isn't a bugfox, it's a tweak to how the descriptions are laid out.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -1856,7 +1856,7 @@
     "id": "LIGHT_BONES",
     "name": "Light Bones",
     "points": 2,
-    "description": "Your bones are very light.  This enables you to move and attack 10% faster, but also reduces your carrying weight by 20% and makes bashing attacks hurt a little more.",
+    "description": "Your bones are very light.  They decrease movement and attack costs for 10%, but also reduce your carrying weight by 20% and makes bashing attacks hurt a little more.",
```
Comment (Qrox) [inline data/json/mutations/mutations.json]:
```suggestion
    "description": "Your bones are very light.  They decrease movement and attack costs by 10%, but also reduce your carrying weight by 20% and make bashing attacks hurt a little more.",
```

File: data/json/mutations/mutations.json [hunk 2]
```diff
@@ -4566,7 +4566,7 @@
     "name": "Hollow Bones",
     "points": 1,
     "mixed_effect": true,
-    "description": "You have Avian Bone Syndrome--your bones are nearly hollow.  Your body is very light as a result, enabling you to move and attack 20% faster, but also frail; you can carry 40% less, and bashing attacks injure you more.",
+    "description": "Your bones are nearly hollow.  They decrease movement and attack costs for 20%, but also reduce your carrying weight by 40%, and bashing attacks injure you more.",
```
Comment (Qrox) [inline data/json/mutations/mutations.json]:
```suggestion
    "description": "Your bones are nearly hollow.  They decrease movement and attack costs by 20%, but also reduce your carrying weight by 40%, and bashing attacks injure you more.",
```

**Comment by I-am-Erk** [comment]
This definitely doesn't close the associated issue.

**Comment by arijust** [comment]
Yes, it does.
These traits don't change speed **directly**, these traits only lower movement costs.

**Comment by I-am-Erk** [comment]
The issue in question is asserting that these mutations aren't stacking correctly. I don't believe it's actually an issue, because it sounds like the op was mistaken on the things they're asserting, but changing the description somewhat laterally doesn't resolve that

**Comment by arijust** [comment]
Should I just close it, in this case?

**Comment by I-am-Erk** [comment]
I'm not sure that a description change is really necessary, but I don't hold myself to be an authority on this one, we could get a few people to weigh in.

**Comment by arijust** [comment]
Ok.

**Comment by kevingranade** [comment]
Yea the linked issue is a misunderstanding, and no need to churn the strings without fixing anything.

**Pull request closed without merging**